### ISSUE TITLE
The parameter in transforms.Normalize() for ImageNet-1K

### ISSUE THREAD

**Issue by Yamato1** [lukemelas/PyTorch-Pretrained-ViT#20]
Excuse me, I want to know what are the parameters in transforms.Normalize(, )  when fine-tune on the ImageNet-1k dataset.
